docs/content/latest/explore/ysql-language-features/tablespaces.md

```diff
@@ -0,0 +1,156 @@
+
+```sql
+CREATE TABLESPACE us_east_tablespace 
+WITH (replica_placement='{"num_replicas": 5, "placement_blocks":
+[{"cloud":"aws","region":"us-east","zone":"us-east-1a","min_num_replicas":1},
+{"cloud":"aws","region":"us-east","zone":"us-east-1b","min_num_replicas":1},
+{"cloud":"aws","region":"us-east","zone":"us-east-1c","min_num_replicas":1}]}');
+```
+
+Even though `us_east_tablespace` shown in the preceding example has been created successfully, a notice is displayed after the execution stating that the additional two replicas are to be placed at the YB-Load balancer’s discretion in  `aws.us-east.us-east-1a`, `aws.us-east.us-east-1b`, and `aws.us-east.us-east-1c`. Based on the load, `aws.us-east.us-east-1a` may have three replicas and the other zones may have one, or it might later change such that `aws.us-east.us-east-1a` and `aws.us-east.us-east-1b` have two replicas each, whereas `aws.us-east.us-east-1c` has one replica. YB-Load balancer always honours the value of `min_num_replicas` and have at least one replica in each cloud.region.zone, and the additional replicas may be moved around based on the cluster load. 
+
+### How to Create a Single-Zone Tablespace
+
+To highlight the difference between multi-zone and single-zone tablespaces, the following example shows how to create a single-zone tablespace:
+
+DEV INPUT REQUIRED
+
+```sql
+CREATE TABLESPACE us_west_1a_tablespace 
+WITH (replica_placement='{"num_replicas": 1, "placement_blocks":
+[{"cloud":"aws","region":"us-west","zone":"us-west-1a","min_num_replicas":1}]}');
+```
+
+If you create a table and use the preceding tablespace, each tablet of the table will have one replica, as per the `num_replicas` value. Since `placement_blocks` contains one placement block where `min_num_replicas` is set to 1, YB-Load balancer will ensure that `aws.us-west.us-west-1a` will have one replica.
+
+## Creating Tables and Indexes in Tablespaces
+
+You can associate new tables and indexes with a corresponding tablespace. This defines the replication factor of the table or index. It also defines how the replicas of the table or index are to be spread across cloud, regions, and zones.
+
+### How to Use a Multi-Zone Tablespace
+
+Using the multi-zone tablespaces defined in [How to Create a Multi-Zone Tablespace](how-to-create-a-multi-zone-tablespace), you can apply the YSQL's `TABLESPACE` option to `CREATE TABLE` and `CREATE INDEX` statements, as follows:
+
+```sql
+CREATE TABLE employees (
+  employee_no integer PRIMARY KEY,
+  name text,
+  department text,
+  change_date date
+)
+TABLESPACE us_west_tablespace;
+```
+
+```sql
+CREATE INDEX employee_no_idx ON employees(employee_no) 
+TABLESPACE us_east_tablespace;
+```
+
+The preceding statements ensure that data in the `employees` and `employee_no_idx` tables is present based on the placement policies specified for `us_west_tablespace` and `us_east_tablespace` respectively.
+
+### How to Use a Single-Zone Tablespace
```

Comment:
Should we flip the order of these 2 sections?

Reply:
@deeps1991 I'm not sure. Karthik wanted to us to add a single-zone information mainly for comparison purposes.  I've just added a sentence that addresses the comparison aspect, I think. Here's what we have now:

### How to Use a Single-Zone Tablespace
There is no difference between creating tables and indexes that use multi-zone tablespaces and those that use single-zone tablespaces. 

Using the single-zone tablespace defined in [How to Create a Single-Zone Tablespace](how-to-create-a-single-zone-tablespace), you can apply the YSQL's `TABLESPACE` option to `CREATE TABLE` and `CREATE INDEX` statements, as follows:

```sql
CREATE TABLE employees (
  employee_no integer PRIMARY KEY,
  name text,
  department text,
  change_date date
)
TABLESPACE us_west_1a_tablespace;
```

```sql
CREATE INDEX employee_no_idx ON employees(employee_no) 
TABLESPACE us_east_1a_tablespace;
```

The preceding statements ensure that data in the `employees` and `employee_no_idx` tables is present based on the placement policies specified for `us_west_1a_tablespace` and `us_east_1a_tablespace` respectively.